app/app.jsx

```diff
@@ -2,14 +2,27 @@ import React from 'react';
 import './scss/style.scss';
 import 'normalize.css';
 
+/* import components */
+import Main from './containers/main.jsx';
+import Nav from './containers/nav.jsx';
+
 class App extends React.Component {
+
    render() {
+
+     const store = this.props.store;
+
       return (
-         <div>
-            Hello World!!!
+         <div className="page">
+           <Nav
+             store={ store.getState() }
```

Comment:
You should only provide the exact props these components need, passing the whole apps state to each component is wrong.

Also, you have given the apps state the name `store`